files/ruby/worker/scripts/docker/bin/deploy.sh

```diff
@@ -1,64 +0,0 @@
-#!/usr/bin/env bash
-
-raw_environment_name=$1
-raw_environment_type=$2
-AWS_ACCESS_KEY_ID=`aws configure get aws_access_key_id`
-AWS_SECRET_ACCESS_KEY=`aws configure get aws_secret_access_key`
-AWS_REGION=`aws configure get region`
-
-function qualified_environment_name {
-  echo $raw_environment_name-$(app_name)-$(app_version)-$raw_environment_type | sed 's/_/-/g' | sed 's/\./-/g'
-}
-
-function qualified_rails_environment_name {
-  if [ $raw_environment_name == prod ]; then echo production; else echo $raw_environment_name; fi
-}
-
-function secret_key_base {
-  echo $(base64 /dev/urandom | head -c 60)
-}
-
-function app_version {
-  cat ./VERSION
-}
-
-function app_name {
-  cat ./APP_NAME
-}
-
-function is_existing_environment {
-  errors=$(eb use $(qualified_environment_name) | grep "ERROR: Environment")
-  if [$errors == ""]; then return 0; else return 1; fi
-}
-
-function qualified_instance_type {
-  if [ $raw_environment_name == 'prod' ]; then echo 't2.medium'; else echo 't2.micro'; fi
-}
-
-#Copy the appropriate Dockerrun file to the root
-cp ./scripts/docker/$raw_environment_name-Dockerrun.aws.json ./Dockerrun.aws.json
-git add Dockerrun.aws.json
-
-#Try to use the current eb environment
-#If it exists deploy to existing environment
-#If not then create a new one (blue/green deployment)
-if is_existing_environment; then
-  echo "Deploy"
-  eb deploy $(qualified_environment_name) --timeout 30 --staged
-else
-  echo "Create"
```

Comment:
Well yes and no. We've noticed that sometimes eb environments will become erratic and no deploys will work or the health will just stay in the red. Being able to kill it and spin up a new one could be helpful. 

Also, spinning up easily would allow the ability to spin up feature environments in the instance of testing things without touching existing environments